charts/retool/values.yaml

```diff
@@ -95,6 +95,8 @@ externalSecrets:
   # Support for legacy external secrets, note this is deprecated in favour of External Secrets Operator: https://github.com/godaddy/kubernetes-external-secrets
   # This mode only allows a single secret name to be provided.
   enabled: false
+  # If External Secrets are in use, determines whether or not to additionally support reading in configuration secrets as kubernetes secrets directly.
```

Comment:
i feel like this param is not completely clear

i like the explanation u made in the description of the PR, maybe add the entire string to the comment here?